EIPS/eip-5094.md

```diff
@@ -0,0 +1,95 @@
+---
+eip: 5094
+title: URI Format for Ethereum Network Switching
+description: A way of representing various network configurations as URLs.
+author: Luc van Kampen (@lucemans), Jakob Helgesson (@svemat01), Joshua Hendrix (@thejoshuahendrix)
+discussions-to: https://ethereum-magicians.org/t/5094-uri-format-for-ethereum-network-switching/9277
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-05-13
+requires: 681, 831
+---
+
+## Abstract
+
+This standard includes all needed information for adding a network to a wallet via URL, by including parameters such as `chainId`, `rpc_url`, `chain_name` and others, such that the network configuration is provided through the URL itself.
+
+## Motivation
+
+As observed with the use of [EIP-681](./eip-681.md) and its implementation in current mobile wallets, transactions can be made, approved, viewed, and used. However, if the wallet is instructed to perform a transaction on a chain they have not yet been configured before, the operation tends to fail.
+
+Although understandable, as the `chain_id` provided makes up only one part of what is required to connect to a network. This EIP aims to introduce a new type of URL for usage with deep-linking, QR, and more, to allow users to seamlessly add new networks to their (for ex. mobile) wallet to then be able to more easily partake in `pay-`, `tx-`, or other Ethereum URL interactions.
+
+As an extension to [EIP-831](./eip-831.md) and neighboring [EIP-681](./eip-681.md) and [EIP-2400](./eip-2400.md), this document aims to standardize the addition of new networks and switching thereof through the means of URLs. User convenience in this case is primary.
+
+Introduction of this EIP is ment to bridge to a safer rpc listing system to be introduced in the near future.
+
+## Specification
+
+### Syntax
+
+Network Switching URLs contain "ethereum" in their schema (protocol) part and are constructed as follows:
+
+    network_add             = erc831_part "@" chain_id [ "/" ] "?" parameters
+    erc831_part             = "ethereum:network-add" 
```

Comment:
I'd recommend splitting this into two parts so that it's more obvious that `add` is part of the ERC-831 `payload` (and not `prefix`.)